glossary.yml

```diff
@@ -14,13 +14,21 @@
       val binne twee en 99.7% val binne drie standaardafwykings. Omgekeerd, in die
       meeste gevalle val 0.3% van die waardes meer as drie standaardafwykings bo of
       onder die gemiddelde.
-      
+  es:
+    term: "regla 68-95-99.7"
+    def: >
+      Expresa el hecho de que el 68% de los valores se encuentran dentro de una
+      [desviación estándar](#standard_deviation) de la [media](#mean), el
+      95% está dentro de dos y el 99,7% está dentro de tres. Por el contrario,
+      aproximadamente el 0,3% de los valores se encuentran más de tres desviaciones
+      estándar por encima o por debajo de la media en la mayoría de los casos.
   fr:
     term: "règle de 68-95-99.7"
     def: >
       Exprime le fait que 68% des valeurs sont localisées à l'intérieur d'un (1)
       [écart-type](#standard_deviation) de la [moyenne](#mean), 95% à l'intérieur de 
-      deux (2) écarts-types, et 99,7% à l'interieur de trois (3) écarts-types. Inversement et dans la plupart des cas, environ 0.3% des valeurs 
+      deux (2) écarts-types, et 99,7% à l'interieur de trois (3) écarts-types.
+      Inversement et dans la plupart des cas, environ 0.3% des valeurs 
```

Comment:
I'd suggest to don't change the formatting of other languages (even though I prefer them as you've done it!)

Reply:
Yes, I realized this later and didn't change any other empty line.